.rubocop.yml

```diff
@@ -0,0 +1,197 @@
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.3
```

Comment:
then I'd get different errors reported ... trying to run on a single version and use that as long as other versions are passing